test/common/basic.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+const assert = require('assert')
+
+// This is really just to see if the common testing framework
+// between NodeJS and Electron works.
+describe('Basic', function () {
+  it('should be always true', function () {
+    assert(true)
+  })
+})
```

Comment:
I would also add the negative test, to check that we get notified of errors too.

describe('Basic', function () {
  it('should be always false', function () {
     assert(false)
   })
 })